tests/ui/or_fun_call.stderr

```diff
@@ -42,38 +42,38 @@ error: use of `unwrap_or` followed by a call to `default`
 LL |     with_default_type.unwrap_or(u64::default());
    |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `with_default_type.unwrap_or_default()`
 
-error: use of `unwrap_or` followed by a function call
-  --> $DIR/or_fun_call.rs:56:14
+error: use of `unwrap_or` followed by a call to `new`
```

Comment:
The "to `new`" here is not optimal, but the suggestion of `unwrap_or_default` is better than `unwrap_or_else(|| vec![])`, IMO.